cogs/admin.py

```diff
@@ -162,6 +162,29 @@ async def shutdown(self, _: Context):
         sleep(1)  # this sleep is there to avoid a Exception in asyncio
         logger.warning("All connections are closed. Hope you will activate me again soon.")
 
+    @command(brief="Toggle command",
+             help="Enables/Disables a command.")
+    @has_guild_permissions(administrator=True)
+    async def toggle(self, ctx: Context, *, cmd):
+        """Enables or disables an existing bot command."""
+        command = self.bot.get_command(cmd)
+        title = "Toggle command"
+
+        if command is None:
+            embed = Embed(title=title,
+                          description=f"Command not found!")
+            await ctx.reply(embed=embed)
+        elif ctx.command == command:
+            embed = Embed(title=title,
+                          description=f"Command not found!")
```

Comment:
Make this a better description like "Can't disable the command to disable the command. You do see the problem don't you?"